src/languages/en.js

```diff
@@ -260,6 +260,7 @@ export default {
         split: ({amount}) => `Split ${amount}`,
         send: ({amount}) => `Send ${amount}`,
         noReimbursableExpenses: 'This report has an invalid amount',
+        pendingConversionMessage: 'Total will update when you\'re back online',
```

Comment:
Shouldn't this be ?

```suggestion
        pendingConversionMessage: 'Total will be updated when you\'re back online',
```

Reply:
Not really, we settled on the copy in this PR.